engine/api/api.go

```diff
@@ -127,6 +128,14 @@ type Configuration struct {
 			Password string `toml:"password"`
 		} `toml:"kafka"`
 	} `toml:"events" comment:"#######################\n CDS Events Settings \n######################"`
+	FeaturesFlipping struct {
+		Izanami struct {
+			ApiURL       string `toml:"api_url"`
+			ClientID     string `toml:"client_id"`
+			ClientSecret string `toml:"client_secret"`
+			Token        string `toml;"token" comment:"Token shared between Izanami and CDS to be able to send webhooks from izanami"`
+		}
+	}
```

Comment:
missing doc for toml configuration, and toml key name for FeaturesFlipping and Izanami